integration

```diff
@@ -250,6 +250,7 @@
   "DeerMaximum/Technische-Alternative-CoE",
   "definitio/ha-rhvoice",
   "definitio/ha-sox",
+  "deler-aziz/fuel_prices_sweden",
```

Comment:
This repository seems abandoned already ?
There has been no movement in 3 months and there are several open PRs.

Reply:
Hi,

All PRs are fixed and merged and the repo is not abandoned